README.md

```diff
@@ -62,6 +62,10 @@ However far you and your partner get, the application should have, at a minimum,
 - [RESTful routing](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/08-rails/mvc-and-restful-routing.md)
 - Deployed application to [Heroku](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/11-deploying-to-heroku.md)
 - A "look and feel" that will make you and your partner happy!
+- Your application must have controller & model tests written
+  - Your model tests should include tests for validations, relationships and any custom methods
+  - We have provided a set of sample tests that you can use as a starting place for the `Driver` and `Passenger` models.  We have also provided a scaffolding for the `Trip` model.  You should feel free to adapt the tests to meet your design.
+  - You will also need to write tests for your controller actions.
```

Comment:
Maybe note that we have stubbed out controller tests for all the RESTful routes, and if they add any custom routes they'll need to go beyond what we've provided.